smoker/src/smoker/smoker/smoker.go

```diff
@@ -24,7 +24,13 @@ var commands map[string]func([]string, backends.Backend, backends.Credential)
 
 // Main function for smoker
 func main() {
+	// TODO handle proper switching via a cli argument
+	startSmoker()
```

Comment:
This is stale from when I decided thought this should be in the same binary, needs to be removed.